provider/gce/environ.go

```diff
@@ -150,14 +152,27 @@ func (e *environ) SetCloudSpec(spec environscloudspec.CloudSpec) error {
 		ClientEmail: credAttrs[credAttrClientEmail],
 		PrivateKey:  []byte(credAttrs[credAttrPrivateKey]),
 	}
+
+	sslVerification := utils.VerifySSLHostnames
+	if spec.SkipTLSVerify {
+		sslVerification = utils.NoVerifySSLHostnames
+	}
+
 	connectionConfig := google.ConnectionConfig{
 		Region:    spec.Region,
 		ProjectID: credential.ProjectID,
+
+		// TODO (Stickupkid): Pass the http.Client through on the construction
+		// of the environ.
+		HTTPClient: utils.GetHTTPClient(sslVerification),
 	}
 
+	// TODO (stickupkid): Pass the context through the method call.
+	ctx := stdcontext.Background()
```

Comment:
This really should be a context.TODO(), same same, but here we are expecting context to be wired through eventually.